gradle/libs.versions.toml

```
@@ -6,6 +6,7 @@ androidx-activity = "1.5.1"
 androidx-autofill = "1.1.0"
 androidx-collection = "1.2.0"
 androidx-compose = "1.4.3"
+androidx-compose-material3 = "1.1.0-rc01"
```

Latest stable release is `1.1.0`:

```suggestion
androidx-compose-material3 = "1.1.0"
```